### PR TITLE
feat: update transaction documentation

### PR DESCRIPTION
**Pull request by ephys**
A branch I wrote a while ago but apparently never sent?

This documents the changes made in Sequelize 7 regarding transactions

### CODE REVIEW

File: docs/other-topics/transactions.md [hunk 1]
```diff
@@ -2,252 +2,241 @@
...
   // The transaction has already been rolled back automatically by Sequelize!
-
 }
 ```
 
-Note that `t.commit()` and `t.rollback()` were not called directly (which is correct).
+### Using save points
 
-### Throw errors to rollback
+In dialects that support save point, you can create a new save point by nesting managed transaction calls:
```
Comment (WikiRik) [inline docs/other-topics/transactions.md:51]: How do people know which dialects support save points?

**Review by WikiRik — CHANGES_REQUESTED** [review]
Since this we've brought back the usage of CLS. Can you update that?

File: docs/other-topics/transactions.md (hunk 1 above)
Comment (ephys) [reply docs/other-topics/transactions.md:51]: We can add a support table like here: https://640374baea7cbb5439a3990a--sequelize-site.netlify.app/docs/v7/querying/operators/#array-operators

Comment (WikiRik) [reply docs/other-topics/transactions.md:51]: That would be nice

Comment (ephys) [reply docs/other-topics/transactions.md:51]: Oh yeah, I forgot about that! Can we do that in the other PR as that support able component is not available in this PR?

Comment (WikiRik) [reply docs/other-topics/transactions.md:51]: That's fine with me

**Comment by WikiRik** [comment]
I forgot about this one, should we revisit this after #330 ? (I'll finish the review of the associations folder soon) Also with https://github.com/sequelize/sequelize/pull/16143 being merged now.

**Comment by ephys** [comment]
Sure! I'll update it with the new info once the main PR is merged

**Comment by ephys** [comment]
@WikiRik CI fails due to an issue with our Netlify deployment that I have not inspected, but the PR is ready to review

**Comment by WikiRik** [comment]
It seems to have deployed fine; https://64989bbcdd20973b67da64a0--sequelize-site.netlify.app/
It's just that the result to the CI failed for some reason, I saw this on the draft build earlier this week as well

**Review by WikiRik — APPROVED** [review]

**Pull request merged**